src/hw12_Parser/BookReader.java

```diff
@@ -0,0 +1,34 @@
+package hw12_Parser;
+
+import java.io.*;
+import java.nio.file.Path;
+import java.util.*;
+
+public class BookReader{
+    private static final Path DIRECTORY_PATH = Path.of("src/hw12_Parser/books"); // path to root package
+
+    // Method for reading the file and getting its content
+    public static String getBookContent(String bookName) throws IOException {
+        Path path = Path.of( DIRECTORY_PATH + "/" + bookName + ".txt");
+
+        BufferedReader reader = new BufferedReader(new FileReader(path.toFile()));
+        StringBuilder content = new StringBuilder();
+        String line;
+
+        while((line = reader.readLine()) != null)
+            content.append(line);
+
+        String bookContent = content.toString().toLowerCase();
+        reader.close();
+
+        return bookContent;
+    }
+
+    // Method for splitting book content to words
+    public static List<String> splitBookToWords(String bookContent){
+        List<String> list = new ArrayList<>(Arrays.asList(bookContent.split("[^A-zА-я']")));
+        list.removeIf(a -> a.length() < 3);
```

Comment:
```suggestion
        list.removeIf(word -> word.length() < 3);
```
it would be better to give more clear names

Reply:
it's better to add comment that short words are excluded, as I just spent 10 minutes to understand what does < 3 mean 😅